src/app/services/wallet.service.ts

```diff
@@ -180,14 +180,21 @@ export class WalletService {
       .reduce((a , b) => a + b, 0));
   }
 
-  private addWallet(wallet) {
-    this.all.first().subscribe(wallets => {
-      wallets.push(wallet);
-      this.saveWallets(wallets);
+  transaction(txid: string): Observable<any> {
```

Comment:
This function apparently it is not used anywhere. Is there any particular reason why it was added?

Reply:
No particular reason. Will be removed in next commit.